sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -769,16 +769,27 @@ class CodegenContext {
       foldFunctions: Seq[String] => String = _.mkString("", ";\n", ";")): String = {
     val blocks = new ArrayBuffer[String]()
     val blockBuilder = new StringBuilder()
+    val defaultMaxLines = 100
+    val maxLines = if (SparkEnv.get != null) {
+      SparkEnv.get.conf.getInt("spark.sql.codegen.expressions.maxCodegenLinesPerFunction",
```

Comment:
This is not following what we are doing for the other SQLConf. I am also thinking if we should just put it into `StaticSQLConf`. Let me check it with others.

Reply:
@gatorsmile Is there any progress?

Reply:
There are multiple related PRs. Maybe we can wait until the reviews are finished there?

Reply:
I see.

Reply:
ping @gatorsmile

Reply:
Can we use bytecode size?

Reply:
Here, we do not know precise bytecode size. Will we use estimated bytecode size based on characters per line?

Reply:
gentle ping @gatorsmile